### DISCUSSION TITLE
Create a Databricks Spark asset

### DISCUSSION THREAD

**Discussion by srggrs** [dagster-io/dagster#11590] [Q&A]
I was wondering how to create a databricks spark asset in dagster using the dagster_databricks step laucher. I can see the step launcher can create an op but not sure what is needed to create an asset.

Thanks in advance.

**Comment by sryza** [comment]
Hi @srggrs - you can use it on an asset in a similar way to how you can use it on an op:

```python
@asset(required_resource_keys={"pyspark_step_launcher"})
def asset1(input1: DataFrame) -> DataFrame:
    ...

defs = Definitions(assets=[asset1], resources={"pyspark_step_launcher": databricks_pyspark_step_launcher.configured(...)})
```
